A/AM/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "AM"
+version = v"13.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://zenodo.org/records/8161261/files/am-13.0.tgz", "1298257dc6a1f50aabc495bc7055e27975f03b67bb28e8f59f812ff39ddd3a36")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/am-13.0/src/
+make am
```

Comment:
Can we build in parallel, to speed up the job?
```suggestion
make -j ${nproc} am COMPILER_GCC=cc
```

Reply:
With the "COMPILER_GCC=cc" flag, the build fails on FreeBSD and Apple systems. The AM codebase uses only gcc for compilation and the default Clang compiler in FreeBSD and Apple seem to fail. The parallel build succeeds on all systems with this flag removed.